vendor_skins/Evercast/CI/before-deploy-osx.sh

```diff
@@ -11,6 +11,7 @@ set -e
 export GIT_HASH=$(git rev-parse --short HEAD)
 export FILE_DATE=$(date +%Y-%m-%d.%H:%M:%S)
 export FILENAME=$FILE_DATE-$GIT_HASH-$TRAVIS_BRANCH-osx.pkg
+export DEPLOY_VERSION=2.5
```

Comment:
Should this be an env var? Updating the script for every release might make updating annoying.